samples/meetings-sidepanel/csharp/Side Panel/Model/configure.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
```

Comment:
Please check and remove if this class is not being used anywhere.

Reply:
This is not used. 
![image](https://user-images.githubusercontent.com/31851992/115445974-2c4a5480-a234-11eb-8f5d-50df5f07708a.png)